cmd/imageproxy/main.go

```diff
@@ -38,6 +38,7 @@ var denyHosts = flag.String("denyHosts", "", "comma separated list of denied rem
 var referrers = flag.String("referrers", "", "comma separated list of allowed referring hosts")
 var includeReferer = flag.Bool("includeReferer", false, "include referer header in remote requests")
 var followRedirects = flag.Bool("followRedirects", true, "follow redirects")
+var maxRedirects = flag.Uint("maxRedirects", 20, "maximum redirection-followings allowed: 0-254 range. 0 is no limit")
```

Comment:
I'm not totally sure what a good default is for this case, I though 20 is ok as it's similar of browser's defaults. We could even go with a 0 to be backward compatible (by default 😂).

Reply:
[the default  in the stdlib is 10](https://cs.opensource.google/go/go/+/master:src/net/http/client.go;l=811-816?q=redirect&ss=go%2Fgo:src%2Fnet%2Fhttp%2F).

Does this actually need to be configurable though?  Does your use-case require a custom number, or you just want *some* limit rather than the current unbounded redirects?  If you just need some limit, I'd rather not expose this as a new flag until it's actually needed.

Reply:
The reason of having it configurable was that we had to do some testing in order to figure out a good value, eventually turns out 10 is ok so [now It's a constant](https://github.com/willnorris/imageproxy/pull/324/commits/c50ac9a1cbf77f85dcb15052bb5b57aaed81ffd2)